lib/sectioned-multi-select.js

```diff
@@ -201,11 +204,10 @@ class SectionedMultiSelect extends PureComponent {
     hideSearch: false,
     selectChildren: false,
     highlightChildren: false,
-    numberOfLines: null,
+    numberOfLines: 1,
```

Comment:
`numberOfLines` prop is used in the item / sub item component. Yours should be named something else e.g `labelNumberOfLines`.